formidable/views.py

```diff
@@ -21,6 +24,86 @@
 from formidable.serializers.presets import PresetsSerializer
 
 
+logger = logging.getLogger(__name__)
+
+
+def extract_function(func_name):
+    """
+    Return a function out of a namespace
+    """
+    func = None
+    if func_name:
+        module, meth_name = func_name.rsplit('.', 1)
+        try:
+            if six.PY3:
+                imported_module = importlib.import_module(module)
+            else:
+                imported_module = importlib.import_module(
+                    module, [meth_name])
+            func = getattr(imported_module, meth_name)
+        except ImportError:
+            logger.error(
+                "An error has occurred impossible to import %s", func_name
+            )
+    return func
+
+
+class CallbackMixin(object):
+    success_callback_settings = ''
+    failure_callback_settings = ''
+    callback_error_message = "An error has occurred with function: `%s`"
+
+    def _call_callback(self, callback):
+        """
+        Tool to simply call the callback function and handle edge-cases.
+        """
+        func = extract_function(callback)
+        # Call function only if existing
+        if not func:
+            return
+        try:
+            func(self.request)
+        except Exception:
```

Comment:
the problem is that if you misconfigure your settings value, you'll only get this exception at runtime, and not before. **Every** form updated will crash, and every update view will throw a 500 - the db transaction will probably be rolled back so your data won't be recorded ; and you'll need a redeployment with a modified settings value to fix this (which could mean a new release, etc).
I'd be sad to see that, even if my form data is valid, that the  ``form.save()`` has been rejected because of an operation that has nothing to do with me, but comes from a "post-update" operation. Let's imagine this as a celery task, *"fire and forget"*...

May I suggest that, in the next iteration of this feature, we could eventually add a boolean settings: ``FORMIDABLE_IGNORE_EXCEPTIONS_ON_CALLBACK``, set to False by default. If set to True, the caught exception would be re-raised.